benchmarks/README.md

```diff
@@ -1,5 +1,11 @@
 # WSO2 Identity Server Performance Results.
 
+In this version of Performance results, the tests were executed against the default 7.0.0 base product without enabling or disabling any additional features.
```

Comment:
This is incorrect README.md to include this info.. It should be 7.0.0/README.md

Reply:
Fixed with debf7d98f992c6a0e295607c1b991e9452f3baa5